### ISSUE TITLE
`Graph.from_geodataframe()` does not copy data columns to the resulting graph

### ISSUE THREAD

**Issue by leehach** [mggg/GerryChain#328]
Using GerryChain 0.2.12

# Expected Behavior

When I run `Graph.from_geodataframe()` on a GeoDataFrame, I expect the data columns to be copied to the node attributes of the resulting graph.

# Actual Behavior

None of the data columns are copied to the node attributes.

# More Information

The `Graph.from_geodataframe()` method has a `cols_to_add` parameter which is explained as follows:

> **cols_to_add** – (optional) The names of the columns that you want to add to the graph as node attributes. By default, all columns are added.

However, the data columns are not copied. If trying to explicitly provide a list of columns to the `cols_to_add` parameter, the method throws an error:

```
Traceback (most recent call last):

  File "<ipython-input-9-0353d16f2ea0>", line 1, in <module>
    graph = Graph.from_geodataframe(gdf, cols_to_add = ["state", "county", "tract"])

TypeError: from_geodataframe() got an unexpected keyword argument 'cols_to_add'
```

The [Getting started with GerryChain](https://gerrychain.readthedocs.io/en/latest/user/quickstart.html) tutorial also supports the idea that copying the columns is expected:

> The `Graph.from_file()` classmethod creates a Graph of the precincts in our shapefile. By default, this method copies all of the data columns from the shapefile’s attribute table to the graph object as node attributes.

# Workaround

As demonstrated on the [Working with Partitions](https://gerrychain.readthedocs.io/en/latest/user/partitions.html), data frame columns can be added as node attributes using `graph.add_data(df)`.

**Comment by zschutzman** [comment]
This was raised in Issue #322 and resolved in PR #324. I guess there hasn't been a new release since then?

**Comment by leehach** [comment]
This is so clear in docs that I thought that might be the issue, but checked that I have latest release. Since I've identified a workaround, I'll probably just wait till the next release rather than use dev.
